src/Components/README.md

```diff
@@ -2,11 +2,33 @@
 
 Aspire components are classic .NET NuGet packages which are designed as highly usable libraries. .NET Aspire components feature rich production-ready telemetry, health checks, configurability, testability, and documentation. For the current state of the components included in this repo and tracked for .NET Aspire's first preview, please check out the [.NET Aspire Components Progress](./Aspire_Components_Progress.md) page.
 
+## Contribution guidelines
+
+We aim to have a diverse set of high quality Aspire components, making it easy to pick from many different technologies when building Aspire apps. We expect to continue to add more components, and we welcome contributions of others, but we explicitly don't want to include every possible component. The set will be gently curated: in order to make sure that components are useful and dependable, we have some broad criteria below for components contributed to dotnet/aspire. These will likely evolve over time based on feedback, but we expect some requirements (such as actively supported) to remain firm:
+
+1. We expect to welcome any components that would have value to Aspire users and align with what Aspire is intended to do, subject to the below.
+2. We don't expect to choose preferred techs. For example, if there are two commonly used providers for database XYZ, we are comfortable with having one component for each. We would like component naming and granularity to be clear enough that customers can make informed decisions. Aspire is agnostic to your choice of cloud provider, too.
+3. We will require that the tech represented by the component is being actively supported. In most cases we expect that it is widely used, although we expect that part will be a judgement call.
+4. Components contributed to dotnet/aspire must meet the same quality and completeness bar of other contributions. ie., we won't have a lower quality bar for experimental or niche components.
+5. Where there's a component that meet the above criteria, but that isn't something we expect to be a high priority for the Aspire committers to maintain, we'll ask for a plan to sustain it (eg., motivated contributors ready to fix bugs in it)
+
+Note: only components that are built from dotnet/aspire will be able to use the Aspire package name prefix. There is no technical barrier to using components built elsewhere, without the Aspire prefix, but currently our idea is that all broadly available Aspire components will live here in dotnet/aspire and have the Aspire package prefix. We welcome feedback on this and all the other principles listed here, though.
```

Comment:
This is a "point in time" limitation. For example, I expect the Azure components to be built from the Azure repo (which will make everyone's lives easier).

Reply:
With the Aspire prefix? So we'll still retain control and expect the prefix, but allow it on packages from certain repos. Basically have trusted packages not trusted package repos.

Reply:
> With the Aspire prefix?

Yes. I don't expect us to rename the Azure components when they move been repos.

Reply:
We only have thoughts beyond the current model, so I'd rather not add any text that looks like we've already committed to how a more distributed model for "official" Aspire components could look in the future.

Reply:
OK I'll merge this as is and no doubt it will evolve